README.md

```diff
@@ -45,38 +45,38 @@ import 'package:system_theme/system_theme.dart';
 
 ### Get system accent color
 
-Use the getter `SystemTheme.accentInstance.accent` to get the system accent color.
+Use the getter `SystemTheme.accentColor.accent` to get the system accent color.
 
 ```dart
-final accentColor = SystemTheme.accentInstance.accent;
+final accentColor = SystemTheme.accentColor.accent;
 ```
 
 To reload the accent colors, use the method `load()`:
 
 ```dart
-await SystemTheme.accentInstance.load();
+await SystemTheme.accentColor.load();
 ```
 
 You can load the colors before running the app, so the colors can't be wrong at runtime:
 
 ```dart
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  await SystemTheme.accentInstance.load();
+  await SystemTheme.accentColor.load();
   runApp(MyApp());
 }
 ```
 
 ### Configure a fallback accent color
 
-To avoid unexpected effects at runtime, it's good to configure a fallback color. A fallback color is used if the system was not able to provide the color
+To avoid unexpected effects at runtime, it's good to configure your own fallback color. The fallback color is used if the system was not able to provide the color.
```

Comment:
```suggestion
To avoid unexpected outcomes at runtime, it's recommended to configure your own fallback color. The fallback color is used if the system is not able to provide the color.
```